CONTRIBUTING.md

```diff
@@ -878,6 +878,29 @@ Incoming pull requests are automatically validated by the CI service. You can wa
 
 ### How To Build & Run Tests On Your Local Machine
 
+#### Add Exchange credentials
+Add exchange credentials and options to `keys.local.json` or as environment variables.
```

Comment:
Can you add the path of `keys.local.json`, it should just be something like `ccxt/keys.local.json` or `root/keys.local.json` or something like that

Also can you add instructions to create `keys.local.json` if the file doesn't exist, I was trying to tell someone to do this once and they were confused because they didnt have a `keys.local.json` file